docs/_sass/faq.scss

```
@@ -27,4 +66,12 @@
     button{
         margin: 20px 0;
     }
+}
+
+@media(max-width:1100px){
+    .container-faq .col-lg-8{
+        padding-left: 4rem;
+        padding-right: 4rem;
```

Let's trim down the padding a bit more for matching this section with the upper ones.

```suggestion
        padding-left: 2rem;
        padding-right: 2rem;
```

---

Yeah Sure.